endoreg_db/views/video_segmentation_views.py

```
@@ -89,9 +89,60 @@ def stream(self, request, pk=None):
+            try:
+                path = Path(vf.active_file.path)
+            except ValueError as exc:
+                raise Http404("No file associated with this video") from exc
+        else:
+            raise Http404("No video file available for this entry")
+
+        if not path.exists():
+            raise Http404("Video file not found on disk")
+
+        mime, _ = mimetypes.guess_type(str(path))
+        response = FileResponse(open(path, 'rb'),
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)